src/azul/chalice.py

```diff
@@ -136,6 +136,8 @@ def spec(self) -> JSON:
             'servers': [{'url': self.self_url('/')}]
         }
 
+    # FIXME: Change self_url to return furl instance
```

Comment:
```suggestion
    # FIXME: Return furl instance
```

since the rest is evident from the context. Same for the other FIXMEs where applicable.